modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormTemplateContextFactoryImpl.java

```diff
@@ -292,6 +292,9 @@ protected void deactivate() {
 
 		templateContext.put("submitLabel", submitLabel);
 
+		templateContext.put(
```

Comment:
This line doesn't have to obey this order, so you can get rid of the surrounding blank lines and abide to alphabetic order.

Reply:
Hey, @boton. It appears that you didn't remove the blank line before the submittable one. Could you please remove it too?

Reply:
I see it was forced by auto SF.